font-changer.py

```diff
@@ -20,9 +20,9 @@
     # if m is a font replace it with the font chosen
     text = text.replace('font-family: \"%s\"' % m, 'font-family: \"%s\"' % font_chosen)
     
-# flag -o to overwrite the original file
-if '-o' in sys.argv:
-    output_file = file_to_change
+# flag -t if you don't want to change the original file
+if '-t' in sys.argv:
```

Comment:
Is this necessary if `output_file = file_to_change` and` file_to_change = "src/index.html"`?

Reply:
I put the flag argument there in case someone doesn't want to change the original file and create a test copy

Reply:
what if the person commits that test file to the repo?

Reply:
Im not sure what the usecase is but, if you think that'd be helpful you should also add that file to `.gitignore`